src/typings.ts

```diff
@@ -15,19 +25,71 @@ export const supportedAuthnMethod = ['passwordless', 'sso', 'credentials'] as co
 export type AuthnMethod = typeof supportedAuthnMethod[number];
 
 export interface ConfigsOptions extends PrivateConfigsOptions {
+  /**
+   * Email used to pre-fill the email field in login or sign up or form.
+   *
+   * Set the default value via {@link MtLinkSdk.init}
+   */
   email?: string;
+  /**
+   * A redirection URL for redirecting a guest back to in the following condition:
+   * - Guest clicks on `Back to [App Name]` button in any Moneytree screen.
+   * - Guest refuses to give consent to access permission in the consent screen.
+   * - Guest logs out from Moneytree via an app with this client id
+   * - Revoke an app's consent from settings screen opened via an app with this client id
```

Comment:
suggest: "id" -> "ID"

also:

Who is the actor in this scenario? All other conditions mention that it's the guest.

Reply:
also the user, I made it explicit. and changed it to ID.